test/services/users.service.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { INestApplication } from '@nestjs/common';
+import { getManager } from 'typeorm'
+import { clearDB, createNestAppInstance } from '../test.helper';
+import { UsersService } from 'src/services/users.service';
+import { Users } from 'src/entities/user.entity';
+const bcrypt = require('bcrypt')
+
+describe('Users Service', () => { 
+  let nestApp: INestApplication;
+  let service: UsersService;
+
+  beforeEach(async () => {
+    await clearDB();
+  })
+
+  beforeAll(async () => {
+    nestApp = await createNestAppInstance();
+    service = nestApp.get<UsersService>(UsersService);
+  })
+
+  describe('Create', () => {
```

Comment:
Also for here as well, I think it would be great if we tested invalid cases as well, like blank params.

Reply:
![image](https://user-images.githubusercontent.com/28055566/167353959-db3adf50-53a6-4f48-a7ba-d5218d69a4a8.png)

Done, I'm a god 😎